### DISCUSSION TITLE
Authentication?

### DISCUSSION THREAD

**Discussion by begunfx** [Tzahi12345/YoutubeDL-Material#937] [Q&A] [answered]
I would like to be able to use this tool outside my home network using a reverse proxy, but it doesn't appear to have any kind of authentication.  Am I missing something?  If not, can it be added?

Thanks!

**Comment by Tzahi12345** [comment, marked as answer]
Yes in the settings you'll find an option called Multi-user mode in the main tab, just enable it and it will ask you to create an admin account. You can then access the Users tab in the settings to create users, roles, disable registration, etc

**Comment by begunfx** [comment]
Perfect.  Thank you!
